1-org/README.md

```diff
@@ -115,7 +115,6 @@ Run `terraform output cloudbuild_project_id` in the `0-bootstrap` folder to see
    adjust your copy paths accordingly.
    ```
    cd gcp-policies
-   git checkout -b main
```

Comment:
Don't we need this if the repo is empty?

Reply:
if the [default branch](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/docs/TROUBLESHOOTING.md#default-branch-setting) is set to main we don't need it.

I checked and it still works to run `git checkout -b main` if the default branch is already set to `main`.

 I will revert this change so that it will work if the default branch is main or master